lib/systems/default.nix

```diff
@@ -24,8 +24,6 @@ rec {
       # Either of these can be losslessly-extracted from `parsed` iff parsing succeeds.
       system = parse.doubleFromSystem final.parsed;
       config = parse.tripleFromSystem final.parsed;
-      # Just a guess, based on `system`
-      platform = platforms.select final;
```

Comment:
Can we provide backcompat here?

Reply:
I guess this wouldn't work with the renames though... Maybe it's not worth it.

Reply:
Ah, true.

Reply:
Oh post merge I realize the back-compat was already not attempting the linux stuff.